mlrun/db/httpdb.py

```diff
@@ -265,8 +265,14 @@ def connect(self, secrets=None):
                     f"warning!, server ({server_cfg['ce_mode']}) and client ({config.ce.mode})"
                     " CE mode don't match"
                 )
-            config.ce.mode = server_cfg.get("ce_mode") or config.ce.mode
-            config.ce.release = server_cfg.get("ce_version") or config.ce.release
+            config.ce.mode = (
+                server_cfg.get("ce", {}).get("mode")
+                or server_cfg.get("ce_mode")
+                or config.ce.mode
+            )
+            config.ce.release = (
+                server_cfg.get("ce", {}).get("release") or config.ce.release
+            )
```

Comment:
```suggestion
            config.ce= (
                server_cfg.get("ce") or config.ce
            )
```

Reply:
This suggestion is for case where you want to overwrite the whole CE struct from the server which is what it seems from what you implemented, if you have cases where you want to have precedence of client configuration for specific attribute inside the `ce` struct than this won't satisfy the requirement